tests/test_setup_config_uncovered.py

```
@@ -2470,3 +2468,44 @@ def test_get_version_info(mocker):
     assert current == "1.0.0"
     assert latest == "1.1.0"
     assert available is True
+
+
+# --- Regression: load_config normalizes empty YAML mappings ---
+
+
+@pytest.mark.configuration
+@pytest.mark.unit
+def test_load_config_normalizes_empty_dict(tmp_path, mocker):
```

Ruff (0.15.12)

[warning] Unused function argument: `tmp_path`

(ARG001)